### DISCUSSION TITLE
How to use tctl to manage leaf cluster

### DISCUSSION THREAD

**Discussion by hydezhao** [gravitational/teleport#13147] [Q&A] [answered]
Hi,

When using `tsh`, we can use `tsh login leaf` to switch to leaf cluster. While when using `tctl`, do we have the equivalent ?
The idea is to use an admin account on root cluster to make some configuration to leaf clusters.

**Comment by webvictim** [comment, marked as answer]
This isn't currently supported, you'll get an error similar to this:

```
ERROR: your credentials are for cluster "orange.teleportdemo.com", please run `tsh login "purple"` to log in to the root cluster
```

If it's something you'd like to see, please raise a feature request: https://github.com/gravitational/teleport/issues/new?assignees=&labels=feature-request&template=feature_request.md
